syncFirebaseSheetScript.gs

```diff
@@ -0,0 +1,80 @@
+/*
+---------------------------------------
+This is a Google Apps Script
+to sync a google sheet
+with a firebase project
+---------------------------------------
+*/
```

Comment:
@drew-haas Add this to your Sheet Macros (Or install it using the project I shared)